### PR TITLE
Create a landing controller and basic structure of home page

### PR DESCRIPTION
**Pull request by CodeSonia**
This PR adds the initial implementation of the landing page

**Changes Made:**
- Created a separate `Landing Controller` to manage the landing page and is separated from other sections
- Created the `index` action to serve the landing page view
- Created the `index.html.erb` file to add the basic structure of the landing page with placeholder content
- Used `link_to` Rails helper to create the buttons with placeholder paths for now
- Updated root route to display index

**Next Steps:**
- Further refine the content of the landing page
- Add styling and visual elements 
- Incorporate CSS

**Screenshot:** (It's pretty _basic_)
image

### CODE REVIEW

File: app/controllers/landing_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class LandingController < ApplicationController
```
Comment (Ryan-Andrews99) [inline app/controllers/landing_controller.rb:1]: Completely for curiosity reasons, but does the `<` indicate inheritance from a superclass?

**Review by Ryan-Andrews99 — APPROVED** [review]
This all looks good to me, it's very clean code! The Ruby/OOP Syntax is very interesting, I can see some similarities in the templating with something like nunjucks. The controller setup in rails is interesting compared to something like express.

File: app/controllers/landing_controller.rb (hunk 1 above)
Comment (CodeSonia) [reply app/controllers/landing_controller.rb:1]: Yesssss spot on! The Application Controller class inherits from ActionController::Base, which is like the foundation for every controllers in Rails.

**Comment by CodeSonia** [comment]
> This all looks good to me, it's very clean code! The Ruby/OOP Syntax is very interesting, I can see some similarities in the templating with something like nunjucks. The controller setup in rails is interesting compared to something like express.

OoOo I've not seen other frameworks, but it's good to know that it's similar to others. Hoping to become competent in rails so I can look at other frameworks with some comfortability!

**Pull request merged**